### PR TITLE
docs: add example customizing unmarshal options

### PR DESCRIPTION
**Pull request by srenatus**
Happy to adjust prose and code as you see fit! 😃

### CODE REVIEW

File: docs/_docs/customizingyourgateway.md [hunk 1]
```diff
@@ -77,6 +77,50 @@ your HTTP endpoints, the response will be pretty-printed.
 Note that this will conflict with any methods having input messages with fields named `pretty`;
 also, this example code does not remove the query parameter `pretty` from further processing.
 
+## Customize unmarshaling per Content-Type
+
+Having different unmarshaling options per Content-Type is possible by wrapping the decoder and
+and passing that to `runtime.WithMarshalerOption`, but not entirely obvious:
```
Comment (johanbrandhorst) [inline docs/_docs/customizingyourgateway.md:83]: Lets remove the commentary and keep it to the implementation.

```suggestion
and passing that to `runtime.WithMarshalerOption`:
```

**Comment by srenatus** [comment]
@johanbrandhorst thanks for the review. Applied your suggestion.

I think this could be something for #1223, too: make this easier. 😉

**Comment by johanbrandhorst** [comment]
Thanks for your contribution! Could you cherry pick this against v2 please?

**Comment by srenatus** [comment]
https://github.com/grpc-ecosystem/grpc-gateway/pull/1341

**Pull request merged**